Source/WebCore/loader/DocumentLoader.cpp

```diff
@@ -2165,7 +2165,7 @@ void DocumentLoader::unregisterReservedServiceWorkerClient()
 #endif
 }
 
-static bool isSandboxingAllowingServiceWorkerFetchHandling(SandboxFlags flags)
+static UNUSED_FUNCTION bool isSandboxingAllowingServiceWorkerFetchHandling(SandboxFlags flags)
```

Comment:
Use `#if ENABLE(SERVICE_WORKER)`.

Reply:
Or, you can move this into DocumentLoader::loadMainResource.
> auto isSandboxingAllowingServiceWorkerFetchHandling = [](SandboxFlags flags) {
>     return !(flags & SandboxOrigin) && !(flags & SandboxScripts);
> }

Reply:
Everyone does so in such a case. I just follow the WebKit style. And, I don't know when to use `UNUSED_FUNCTION`.